src/Polly.Core/Builder/ResilienceStrategyBuilderOptions.cs

```diff
@@ -18,4 +19,10 @@ public class ResilienceStrategyBuilderOptions
     /// Gets the custom properties attached to builder options.
     /// </summary>
     public ResilienceProperties Properties { get; } = new();
+
+    /// <summary>
+    /// Gets or sets an instance of <see cref="TelemetryFactory"/>.
+    /// </summary>
+    [Required]
```

Comment:
Is this required if it has a default implementation?

Reply:
Just in case someone set's it to `null` explicitly.